### DISCUSSION TITLE
Interactive Preview

### DISCUSSION THREAD

**Discussion by metamountain** [Fannovel16/comfyui_controlnet_aux#212] [Ideas]

**Comment by freedomtheworld** [comment]
My computer（mbp m2 ） has this problem after installation with the following code. Please help me.

Keywords: SC ControlNet, Differential Diffusion, default_image_only_indicator, FreeU_Advanced

Feature News:

[SC ControlNet] ControlNet feature for Stable Cascade has been added to ComfyUI.
Issue News:

[FreeU_Advanced] The compatibility patch for "FreeU_Advanced" has been made regarding the "default_image_only_indicator".
[default_image_only_indicator] In the latest ComfyUI, there's an error called "default_image_only_indicator" due to compatibility issues with the old "FreeU_Advanced" custom node.
ComfyUI: 2079[062483](2024-03-21)
Manager: V2.10.2
